wiki/Guides/Common_modding_mistakes/es.md

```diff
@@ -0,0 +1,178 @@
+
+---
+
+```md
+00:00:000 - Borra este slider
+00:01:000 - Añade círculo.
+00:02:000 - Reduce el espacio aquí.
+00:02:000 - Haz que fluya mejor.
+00:04:000 - Ctrl+G
+00:05:000 - Mueve a (100:200)
+00:06:000 - NC
+```
+
+**A menos que el contexto sea claro, indica siempre por qué crees que se deberian de hacer cambios.** Dejar al mapper pensando podría llevar a malentendidos, y por tanto impedir que el mapa pueda mejorarse.
+
+---
+
+```md
+00:00:000 - Igual que #72727
+00:01:000 - Igual que #72727
+00:02:000 - Igual que #72727
+```
+
+**Para problemas recurrentes, considera agruparlos en un bloque general con un par de ejemplos.** Si aún así piensas que mencionarlo todo es absolutamente necesario, seguramente signifique que el mapper no sabe lo que hace o bien tu explicación no es lo suficientemente buena.
+
+---
+
+```md
+00:00:000 - Estos objetos están puestos de manera rara y puede que no sea clasificable.
+00:01:000 - Este espaciado no es clasificable.
+00:02:000 - Creo que este ritmo no es clasificable.
+```
+
+**Evita decir o sugerir que algo no es clasificable a menos que los [criterios de clasificación](/wiki/Ranking_Criteria) explícitamente diga que lo es.** Esto puede causar mucha confusión para los nuevos mappers. Si algo no es clasificable, significa que no puede seguir el [procedimiento de clasificación](/wiki/Beatmap_ranking_procedure), sin excepciones. Ten cuidado usando esta frase.
+
+---
+
+```md
+00:00:000 - lol
+00:01:000 - esto se ve estúpido
+00:02:000 - wow xD
+00:03:000 - no hagas esto
+```
+
+**Cuando mencionas un problema, sé claro y formal para evitar malentendidos.** Siempre deberías exponer cuál es el problema, incluso si parece obvio. 
+
+---
+
+```md
+00:00:000 - ¿Por qué está tan curvado este slider?
+00:01:000 - Aquí está otra vez.
+00:02:000 - Aparece en el kiai también.
+```
+
+**Si algo parece estar por todo el mapa, puede que haya sido intencional.** En estos casos, o bien explica en detalle por qué crees que está mal, o asegúrate de que aparecen en partes similares de la canción. Si són inconsistentes y no parece haber patrón alguno, estaría bien preguntar por ello.
+
+---
+
+```md
+En general, las notas deberían estar mapeadas a distintos sonidos de la canción para poder reflejarla con exactitud. Mantener la densidad del ritmo con la intensidad de la canción es /.../
+```
+
+**Siempre incluye al menos un ejemplo y una línea de tiempo como mínimo.** Sin dar ejemplos en el mapa que estás modeando, el mapper seguramente esté confundido de dónde y qué problemas estás hablando. Como modder a veces puedes acabar iendo por las ramas, olvidándote de mejorar el mapa en el proceso.
+
+---
+
+```md
+El soporte para la resolución panorámica es inconsistente y tu mapa no tiene guin gráfico, por lo tanto desactívalo para todas las dificultades.
```

Comment:
```suggestion
El soporte para la resolución panorámica es inconsistente y tu mapa no tiene storyboard, por lo tanto desactívalo para todas las dificultades.
```

Reply:
For this one, [I based myself on how the wiki translated Storyboard to](https://osu.ppy.sh/wiki/es/Storyboard), though that was only done on the title. Unsure whether that was intended, though I'll apply this one being what's most commonly referred to regardless of the language.